search-app/src/cmr/search/services/autocomplete_service.clj

```diff
@@ -16,7 +16,14 @@
   (if-not (empty? user-acls)
     (gc/or-conds [public-collections-condition
                   (gc/or-conds
-                   (map #(qm/text-condition :permitted-group-ids %) user-acls))])
+                   ; case insensitive pattern condition to allow searching through
+                   ; concatenated group names
+                   (map (fn [acl-id]
+                          (qm/string-condition :permitted-group-ids
+                                               (format "*%s*" acl-id)
+                                               :false
+                                               :true))
+                        (into user-acls ["registered"])))])
```

Comment:
are you assuming if any "user-acls" are passed, that all registered is automatically allowed?  If so this is incorrect.

Reply:
I was - I'll fix this